frontend/config.py

```diff
@@ -27,3 +27,12 @@ class Config(object):
     MYSQL_DATABASE = '{MYSQL_DATABASE}'.format(**os.environ)
 
     SUNNAH_COM_API_KEY = '{SUNNAH_COM_API_KEY}'.format(**os.environ)
+
+    # Flask Mail 
+    MAIL_DEAFULT_SENDER = '{MAIL_DEAFULT_SENDER}'.format(**os.environ)
```

Comment:
can you please update the .env.local.sample file and update the README as well for these new settings?

Reply:
.env.local.sample and ReadMe file has been updated